docs/KNOWNISSUES.md

```diff
@@ -2,138 +2,119 @@
 
 ## Known Issues for PowerShell on Non-Windows Platforms
 
-The first Alpha release of PowerShell on Linux and macOS is mostly functional but
-does have some significant limitations and usability issues.
+Alpha releases of PowerShell on Linux and macOS are mostly functional but do have some significant limitations and usability issues.
```

Comment:
Given, we are targeting cross-plat as a major focus for Beta1, I think the above statement would not apply to Beta1.
It would be good to link a query to see those issues (closed and open both) for community to see progress and up vote on.